onnxruntime/contrib_ops/cuda/bert/longformer_attention_impl.cu

```diff
@@ -382,8 +388,8 @@ bool launchSoftmaxKernel(
   const int* global_count = reinterpret_cast<const int*>(pinned_buffer);
 
   bool is_fp16 = (element_size == 2);
-  void* scratch1 = reinterpret_cast<char*>(workspace);
-  char* scratch2 = (char*)scratch1 + GetScratch1Size(element_size, batch_size, num_heads, sequence_length, window);
+  char* scratch1 = reinterpret_cast<char*>(workspace);
```

Comment:
curious why replace void* with char*?

Reply:
save some typing of reinterpret_cast below